docs/tutorials/Canopy/soil_canopy_tutorial.jl

```diff
@@ -109,6 +110,67 @@ include(
 # Define the parameters for the soil model and provide them to the model 
 # parameters struct:
 
+# Heterotrophic respiration parameters
+P_sfc = FT(101e3)
+ν = FT(0.556)
+θ_a100 = FT(0.1816)
+D_ref = FT(1.39e-5)
+b = FT(4.547)
+D_liq = FT(3.17)
+α_sx = FT(194e3)
+Ea_sx = FT(61e3)
+kM_sx = FT(5e-3)
+kM_o2 = FT(0.004)
+O2_a = FT(0.209)
+D_oa = FT(1.67)
+p_sx = FT(0.024)
+
+soilco2_type = Soil.Biogeochemistry.SoilCO2Model{FT}
+
+soilco2_ps = SoilCO2ModelParameters{FT}(;
+    P_sfc = P_sfc,
+    ν = ν,
+    θ_a100 = θ_a100,
+    D_ref = D_ref,
+    b = b,
+    D_liq = D_liq,
+    α_sx = α_sx,
+    Ea_sx = Ea_sx,
+    kM_sx = kM_sx,
+    kM_o2 = kM_o2,
+    O2_a = O2_a,
+    D_oa = D_oa,
+    p_sx = p_sx,
+    earth_param_set = earth_param_set,
+);
+
+# soil microbes args
```

Comment:
Could you explain a little what you are specifying here? (as this is a tutorial)

also check out the rendered docs. Usually you need a `;` after the last line to prevent it from outputing into the rendered tutorial.